components/movementsensor/adxl345/adxl345.go

```diff
@@ -189,10 +235,90 @@ func NewAdxl345(
 			}
 		}
 	})
-
+	sensor.interruptsFound = make(map[string]int)
+	sensor.readInterrupts(sensor.cancelContext)
+	err = sensor.configureInterruptRegisters(ctx)
+	if err != nil {
+		sensor.cancelFunc()
+		return nil, err
+	}
+	ticksChan := make(chan board.Tick)
+	if sensor.echoInterrupt1 != nil {
+		sensor.startInterruptPolling(sensor.echoInterrupt1, ticksChan)
+	}
+	if sensor.echoInterrupt2 != nil {
+		sensor.startInterruptPolling(sensor.echoInterrupt2, ticksChan)
```

Comment:
We're using the same channel on two different interrupts, and consuming items from it in two different goroutines. Could we get by with a single goroutine for both interrupts? If not, do we need a second channel for the other one? This feels strange to me, though I don't think it will lead to incorrect behavior (just misleading architecture).